backend/models/GpsSettings.js

```
@@ -0,0 +1,191 @@
+            ELSE last_permission_granted 
+          END,
+          updated_at = NOW()
+      WHERE livreur_id = $1
+      RETURNING *
+    `;
+    
+    const result = await db.query(query, [livreur_id, enabled, permission_granted]);
+    
+    if (result.rows.length === 0) {
+      // Créer si n'existe pas
+      return this.createOrUpdate({ 
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)